tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1

```diff
@@ -0,0 +1,264 @@
+<#
+  QuickStart script for setting up Azure for AKS Edge Essentials and deploying the same on the Windows device
+#>
+param(
+    [ValidateNotNullOrEmpty()]
+    [String] $SubscriptionId,
+    [ValidateNotNullOrEmpty()]
+    [String] $TenantId,
+    [ValidateNotNullOrEmpty()]
+    [String] $Location,
+    [Switch] $UseK8s,
+    [string] $Tag
+)
+#Requires -RunAsAdministrator
+New-Variable -Name gAksEdgeQuickStartVersion -Value "1.0.231016.1400" -Option Constant -ErrorAction SilentlyContinue
+
+New-Variable -Option Constant -ErrorAction SilentlyContinue -Name arcLocations -Value @(
+    "westeurope", "eastus", "westcentralus", "southcentralus", "southeastasia", "uksouth",
+    "eastus2", "westus2", "australiaeast", "northeurope", "francecentral", "centralus",
+    "westus", "northcentralus", "koreacentral", "japaneast", "eastasia", "westus3",
+    "canadacentral", "eastus2euap"
+)
+
+if (! [Environment]::Is64BitProcess) {
+    Write-Host "Error: Run this in 64bit Powershell session" -ForegroundColor Red
+    exit -1
+}
+#Validate inputs
+$skipAzureArc = $false
+if ($arcLocations -inotcontains $Location) {
+    Write-Host "Error: Location $Location is not supported for Azure Arc" -ForegroundColor Red
+    Write-Host "Supported Locations : $arcLocations"
+    exit -1
+}
+
+$installDir = $((Get-Location).Path)
+$productName = "AKS Edge Essentials - K3s"
+$networkplugin = "flannel"
+if ($UseK8s) {
+    $productName ="AKS Edge Essentials - K8s"
+    $networkplugin = "calico"
+}
+
+# Here string for the json content
+$aideuserConfig = @"
+{
+    "SchemaVersion": "1.1",
+    "Version": "1.0",
+    "AksEdgeProduct": "$productName",
+    "AksEdgeProductUrl": "",
+    "Azure": {
+        "SubscriptionName": "",
+        "SubscriptionId": "$SubscriptionId",
+        "TenantId": "$TenantId",
+        "ResourceGroupName": "angop-test-aksedge-rg",
+        "ServicePrincipalName": "angop-test-aksedge-sp",
+        "Location": "$Location",
+        "CustomLocationOID":"",
+        "Auth":{
```

Comment:
@parameshbabu i think we wanted to remove the requirement of service Principal - Could you please further advice on how to proceed?

Reply:
Lets take this offline and explore. Not gate this commit on this.